docs/source/release/v3.7.0/muon.rst

```diff
@@ -35,6 +35,7 @@ Muon Analysis
 
 - When the window is resized, all widgets within the window should now resize with it. This enables the interface to be used on smaller screens. `#15382 <https://github.com/mantidproject/mantid/pull/15832>`_
 - "Plot/Remove guess" now deals correctly with the case when a new run is loaded. `#15872 <https://github.com/mantidproject/mantid/pull/15872>`_
+- By default, previous fits now remain on the graph when a new run is loaded, to enable easy comparison between datasets. They can be removed with the "Clear fit curves" option. `#16018 <https://github.com/mantidproject/mantid/pull/16018>`_
```

Comment:
This does not describe the functionality as you described in the test, which implies it is not by default, but only if you set *New plot policy* to *Use previous window*.